src/data/Model.test.js

```
@@ -0,0 +1,223 @@
+            test('returns false', () => {
+                expect(Model.isDryModel(input)).toBe(false);
+            });
+        });
+    }
+});
+
+describe('Model.withName()', () => {
+    test('overrides the name of the model', () => {
+        class RenamedModel extends Model {
+            static {
+                this.withName('AnotherName');
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)